index.js

```diff
@@ -14,20 +14,24 @@ async function asanaOperations(
       defaultHeaders: { 'asana-enable': 'new-sections,string_ids' },
       logAsanaChangeWarnings: false
     }).useAccessToken(asanaPAT);
-    if (sectionName) {
-      let project = await client.sections.findByProject(projectId);
-      if (project) {
-        let requiredSection = project.find(data => data.name === sectionName);
-        if (requiredSection) {
-          await client.sections.addTask(requiredSection.gid, { task: taskId });
-          core.info('Moved to: ' + requiredSection.name);
+
+    if (sectionName && projectName) {
+      let targetProject = await client.tasks.findById(taskId)
```

Comment:
Is the plan that there will be two separate github actions, one for the Core Auto sprint board, and another for the Integrations sprint board? I think this will probably work if that's the case, but it still feels like there's a better solution. Is it possible to pass multiple target projects, and loop over them?

Reply:
Great catch, I'll have to look, how we can pass array of project and section pairs.

Reply:
@iamalisajid helped me to find a solution for it. Only strings can be used as arguments so using an array is not possible we can use JSON string to pass multiple values as suggested [here](https://github.community/t5/GitHub-Actions/Can-action-inputs-be-arrays/td-p/33776). I think there should be a JSON array of objects having `target-project` and `target-section`. A project may have a different `target-section` as compared to other projects in this way this solution will be more generic. @katehedgpeth need your suggestion to implement it.

```yml
targets: '[{"project": "Core Auto Dev - Sprint
", "section": "In Review"}, {"project": "Integrations Dev - Sprint
", "section": "In Review"}]'
```

Reply:
that sounds like it could work, and it would give us more flexibility if different sprint boards use different labels. give it a shot!